regression/cbmc/Float-equality1/test_no_equality.desc

```diff
@@ -0,0 +1,9 @@
+CORE
```

Comment:
The SMT-backend is broken, and this test again shows this. Add `broken-smt-backend`. Preferably also to the `KNOWNBUG` test.